.github/workflows/ci.yml

```diff
@@ -156,6 +156,7 @@ jobs:
           spec: |
             ./cypress/e2e/sampleTablePage.cy.js
             ./cypress/e2e/editPage.cy.js
+            ./cypress/e2e/equipment.cy.js
```

Comment:
Oh god, turn it back off!!

Reply:
I think I fixed it now 😅. And rebased to squash some commits so it looks like I make fewer mistakes in getting this to work.

So it turned out the batch tests had some of the same errors: within() blocks that ran asynchronously and caused race conditions that only showed up when running the tests in CI. 

I decided most all of those within() blocks weren't needed anyways, and deleted most of them from all our tests- but, it turned out I was a little overzealous and lots of things broke! 

For future reference, cypress commands are always asynchronous, but they are mostly completed in order. `within()` causes this not to be true, leading to issues. We were using `within` to select html nodes inside of other nodes we had located, like this:
```js
cy.get(".vs__dropdown-menu").within(() => {
  cy.contains(".badge", "baseA").click();
});
 ``` 
This is now replaced with simply:
```js
cy.get(".vs__dropdown-menu").contains(".badge", "baseA").click();
```

And the slightly trickier case:
```js
cy.get(".table").within(() => {
  cy.get("td.table-item-id .formatted-item-name").click();
});
```
replaced with:
```js
cy.get(".table").find("td.table-item-id .formatted-item-name").click();
```
Note that in this case the `get()` inside the `within()` block has to be replaced with `find()`. This is because `get()` always searches the whole document when it is not used in within(), while `find()` only searches relative to the node it is chained off.

A general rule is that `within()` probably doesn't need to be used when there is only one command/chain of commands inside its callback. It should be used if multiple actions need to be taken within a selected node.

Reply:
Nice work, thanks for the breakdown

I actually think that now we can run all our specs again, we can remove the spec list both here and for chrome? Should make it easier to introduce more e2e tests